inotify/inotify_linux_test.go

```diff
@@ -105,3 +109,31 @@ func TestInotifyClose(t *testing.T) {
 		t.Fatal("expected error on Watch() after Close(), got nil")
 	}
 }
+
+func TestInotifyFdLeak(t *testing.T) {
+	watcher, _ := NewWatcher()
+	defer watcher.Close()
+
+	child := exec.Command("sleep", "1")
```

Comment:
is `"`1`" enough? we could bump it up as we are calling SIGTERM right?

Reply:
The test run time increase is barely noticeable, but if I can bump the value to be safe. Would 10 be ok?